[index management] Move tests from FTR to scout

mattkime opened the pull request:
## Summary

Moving index management tests from FTR to scout. Scout hates it when the data-test-subj value has been used more than once so there are some changes to address that.

<!--ONMERGE {"backportTargets":["8.19","9.1","9.2"]} ONMERGE-->

mattkime committed:
home_page test port

mattkime committed:
home_page test port

kibanamachine committed:
Changes from node scripts/lint_ts_projects --fix

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/360819)
* Commit: 36d966a593d600f6b37537ad71eb36b1b8aa6971

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/360819#019a7be8-7647-4171-98f8-778e23f86595)

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/360819","number":360819,"commit":"36d966a593d600f6b37537ad71eb36b1b8aa6971"}],"number":360819}
buildkite-pr-comment-->

mattkime committed:
fix lint issues

mattkime committed:
fix lint issues

kibanamachine committed:
Changes from node scripts/eslint_all_files --no-cache --fix

mattkime committed:
make data-test-subj's unique

mattkime committed:
Merge branch 'main' into index_mgmnt_playwright

mattkime committed:
Merge branch 'unique_create_index_btn_data_test_subj' into index_mgmnt_playwright

mattkime committed:
Merge branch 'main' into index_mgmnt_playwright

mattkime committed:
functional test fixes

mattkime committed:
Merge branch 'index_mgmnt_playwright' of github.com:elastic/kibana into index_mgmnt_playwright

elasticmachine commented:
## :broken_heart: Build Failed
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/361386)
* Commit: 578416959cee0e9d9bc7b499dfe6ff62babc559e

### Failed CI Steps
* [Linting](https://buildkite.com/elastic/kibana-pull-request/builds/361386#019a829c-85a1-499a-9cb1-93f8aa76d3c5)

### History
* :broken_heart: [Build #361128](https://buildkite.com/elastic/kibana-pull-request/builds/361128) failed 57f2ef80db55768bdc6ffaf28c0281585302efb4
* :broken_heart: [Build #361065](https://buildkite.com/elastic/kibana-pull-request/builds/361065) failed b32f61fb5b7829710b8735ca3c7fb12437b239e6
* :broken_heart: [Build #360823](https://buildkite.com/elastic/kibana-pull-request/builds/360823) failed f3b31407dc03e78909b4553c0c45e996b54db936
* :broken_heart: [Build #360819](https://buildkite.com/elastic/kibana-pull-request/builds/360819) failed 36d966a593d600f6b37537ad71eb36b1b8aa6971

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/361386","number":361386,"commit":"578416959cee0e9d9bc7b499dfe6ff62babc559e"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/361128","number":361128,"commit":"57f2ef80db55768bdc6ffaf28c0281585302efb4"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/361065","number":361065,"commit":"b32f61fb5b7829710b8735ca3c7fb12437b239e6"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/360823","number":360823,"commit":"f3b31407dc03e78909b4553c0c45e996b54db936"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/360819","number":360819,"commit":"36d966a593d600f6b37537ad71eb36b1b8aa6971"}],"number":361386}
buildkite-pr-comment-->

mattkime committed:
remove page.pause

mattkime committed:
fix jest tests

mattkime committed:
remove FTR tests that were ported

mattkime committed:
cleanup and refer to tabs by dataTestSubj instead of position

mattkime committed:
use custom role

mattkime committed:
use custom role

mattkime committed:
remove todo

elasticmachine commented:
## :broken_heart: Build Failed
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/361524)
* Commit: 6014af97f2f00ba7d72f85661288ce91bd5a1a32

### Failed CI Steps
* [Linting (with types)](https://buildkite.com/elastic/kibana-pull-request/builds/361524#019a83a8-2c52-4a90-8952-f9a382f4617b)

### History
* :broken_heart: [Build #361404](https://buildkite.com/elastic/kibana-pull-request/builds/361404) failed 5c2671bd4a2027fcfc584419a01eb30895da3cd4
* :broken_heart: [Build #361386](https://buildkite.com/elastic/kibana-pull-request/builds/361386) failed 578416959cee0e9d9bc7b499dfe6ff62babc559e
* :broken_heart: [Build #361128](https://buildkite.com/elastic/kibana-pull-request/builds/361128) failed 57f2ef80db55768bdc6ffaf28c0281585302efb4
* :broken_heart: [Build #361065](https://buildkite.com/elastic/kibana-pull-request/builds/361065) failed b32f61fb5b7829710b8735ca3c7fb12437b239e6
* :broken_heart: [Build #360823](https://buildkite.com/elastic/kibana-pull-request/builds/360823) failed f3b31407dc03e78909b4553c0c45e996b54db936

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/361524","number":361524,"commit":"6014af97f2f00ba7d72f85661288ce91bd5a1a32"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/361404","number":361404,"commit":"5c2671bd4a2027fcfc584419a01eb30895da3cd4"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/361386","number":361386,"commit":"578416959cee0e9d9bc7b499dfe6ff62babc559e"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/361128","number":361128,"commit":"57f2ef80db55768bdc6ffaf28c0281585302efb4"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/361065","number":361065,"commit":"b32f61fb5b7829710b8735ca3c7fb12437b239e6"}],"number":361524}
buildkite-pr-comment-->

mattkime committed:
await promise

mattkime committed:
fix jest test

mattkime committed:
type fix

mattkime committed:
scout test fixes

mattkime committed:
type fix

mattkime committed:
remove comments

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/361604)
* Commit: d43f4d5ccce1498203c95999f4ac36f98f3a10db

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/1fdcadd16b87f4ae211639135a1f550da975c5a4) | [after](https://github.com/elastic/kibana/commit/d43f4d5ccce1498203c95999f4ac36f98f3a10db) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 705.5KB | 705.7KB | +219.0B |

<details><summary>Unknown metric groups</summary>

#### ESLint disabled in files
| id | [before](https://github.com/elastic/kibana/commit/1fdcadd16b87f4ae211639135a1f550da975c5a4) | [after](https://github.com/elastic/kibana/commit/d43f4d5ccce1498203c95999f4ac36f98f3a10db) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 3 | 4 | +1 |

#### ESLint disabled line counts
| id | [before](https://github.com/elastic/kibana/commit/1fdcadd16b87f4ae211639135a1f550da975c5a4) | [after](https://github.com/elastic/kibana/commit/d43f4d5ccce1498203c95999f4ac36f98f3a10db) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 16 | 17 | +1 |

#### Total ESLint disabled count
| id | [before](https://github.com/elastic/kibana/commit/1fdcadd16b87f4ae211639135a1f550da975c5a4) | [after](https://github.com/elastic/kibana/commit/d43f4d5ccce1498203c95999f4ac36f98f3a10db) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 19 | 21 | +2 |


</details>


### History
* :broken_heart: [Build #361586](https://buildkite.com/elastic/kibana-pull-request/builds/361586) failed 12b0f03012a7b42dc5925c614c0449c65574e45b
* :broken_heart: [Build #361555](https://buildkite.com/elastic/kibana-pull-request/builds/361555) failed c43760d0bc7739901a1e64338a92629b7cd1e91d
* :broken_heart: [Build #361551](https://buildkite.com/elastic/kibana-pull-request/builds/361551) failed 30f5383781e1ab99d37f88e2898d51322bff8f66
* :broken_heart: [Build #361535](https://buildkite.com/elastic/kibana-pull-request/builds/361535) failed e49993027261e10c22f9aa43d563648ca79dd4f3
* :broken_heart: [Build #361524](https://buildkite.com/elastic/kibana-pull-request/builds/361524) failed 6014af97f2f00ba7d72f85661288ce91bd5a1a32

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/361604","number":361604,"commit":"d43f4d5ccce1498203c95999f4ac36f98f3a10db"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/361586","number":361586,"commit":"12b0f03012a7b42dc5925c614c0449c65574e45b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/361555","number":361555,"commit":"c43760d0bc7739901a1e64338a92629b7cd1e91d"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/361551","number":361551,"commit":"30f5383781e1ab99d37f88e2898d51322bff8f66"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/361535","number":361535,"commit":"e49993027261e10c22f9aa43d563648ca79dd4f3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/361524","number":361524,"commit":"6014af97f2f00ba7d72f85661288ce91bd5a1a32"}],"number":361604}
buildkite-pr-comment-->

mattkime committed:
improve user privs

mattkime committed:
move some logic to pageObject to be shared

mattkime committed:
consolidate some logic into pageObject

mattkime committed:
Merge branch 'main' into index_mgmnt_playwright

mattkime committed:
better auth integration

mattkime committed:
add custom role file

mattkime committed:
Merge branch 'main' into index_mgmnt_playwright

elasticmachine commented:
Pinging @elastic/kibana-management (Team:Kibana Management)

mattkime committed:
Merge branch 'main' into index_mgmnt_playwright

kibanamachine committed:
Changes from node scripts/regenerate_moon_projects.js --update

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/364662)
* Commit: 4599facf23803eaaca80ccfc74e6abe27f50e116

### Failed CI Steps
* [FTR Configs #22](https://buildkite.com/elastic/kibana-pull-request/builds/364662#019ab918-241b-4b96-816d-20c31cb03097)
* [FTR Configs #54](https://buildkite.com/elastic/kibana-pull-request/builds/364662#019ab918-2441-4f73-97f4-ec0f410b2f2f)
* [FTR Configs #54](https://buildkite.com/elastic/kibana-pull-request/builds/364662#019ab932-7390-441d-afd0-0a0b7416c12d)
* [FTR Configs #94](https://buildkite.com/elastic/kibana-pull-request/builds/364662#019ab918-246d-42d5-9fe1-a590046e9712)
* [FTR Configs #122](https://buildkite.com/elastic/kibana-pull-request/builds/364662#019ab918-248e-49d1-b022-d5aa28edeed8)
* [FTR Configs #122](https://buildkite.com/elastic/kibana-pull-request/builds/364662#019ab935-1067-4bcc-8ac4-35d0dcd8592b)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/364662#019ab918-241b-4b96-816d-20c31cb03097) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/364662/jobs/019ab918-241b-4b96-816d-20c31cb03097/artifacts/019ab93d-ce51-42a2-9f0f-5b7d924e4058) FTR Configs #<span></span>22 / Index Management app Index Management: index templates tab Index template tab index template modification can modify ignore_above, ignore_malformed, ignore_dynamic_beyond_limit, subobjects and timestamp format in an index template with logsdb index mode
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/364662#019ab918-2441-4f73-97f4-ec0f410b2f2f) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/364662/jobs/019ab918-2441-4f73-97f4-ec0f410b2f2f/artifacts/019ab932-5ca4-4f7a-9dd3-f1d09554908f) FTR Configs #<span></span>54 / Serverless Common UI - Management Index Management Index Templates Modify index template can modify ignore_above, ignore_malformed, ignore_dynamic_beyond_limit, subobjects and timestamp format in an index template with logsdb index mode
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/364662#019ab918-246d-42d5-9fe1-a590046e9712) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/364662/jobs/019ab918-246d-42d5-9fe1-a590046e9712/artifacts/019ab93c-91ff-4899-adfc-01f3bddb1cc7) FTR Configs #<span></span>94 / Serverless Common UI - Management Index Management Index Templates Modify index template can modify ignore_above, ignore_malformed, ignore_dynamic_beyond_limit, subobjects and timestamp format in an index template with logsdb index mode
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/364662#019ab918-248e-49d1-b022-d5aa28edeed8) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/364662/jobs/019ab918-248e-49d1-b022-d5aa28edeed8/artifacts/019ab934-ea1a-4f93-b1ab-a3292ee339d1) FTR Configs #<span></span>122 / Serverless Common UI - Management Index Management Index Templates Modify index template can modify ignore_above, ignore_malformed, ignore_dynamic_beyond_limit, subobjects and timestamp format in an index template with logsdb index mode
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/364662#019ab932-7390-441d-afd0-0a0b7416c12d) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/364662/jobs/019ab932-7390-441d-afd0-0a0b7416c12d/artifacts/019ab93d-5c51-4fc4-aca0-2a733ed3bff5) FTR Configs #<span></span>54 / Serverless Common UI - Management Index Management Index Templates Modify index template can modify ignore_above, ignore_malformed, ignore_dynamic_beyond_limit, subobjects and timestamp format in an index template with logsdb index mode
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/364662#019ab935-1067-4bcc-8ac4-35d0dcd8592b) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/364662/jobs/019ab935-1067-4bcc-8ac4-35d0dcd8592b/artifacts/019ab941-058f-41d3-ada5-2b33f50cd90c) FTR Configs #<span></span>122 / Serverless Common UI - Management Index Management Index Templates Modify index template can modify ignore_above, ignore_malformed, ignore_dynamic_beyond_limit, subobjects and timestamp format in an index template with logsdb index mode

### History
* :green_heart: [Build #361604](https://buildkite.com/elastic/kibana-pull-request/builds/361604) succeeded d43f4d5ccce1498203c95999f4ac36f98f3a10db
* :broken_heart: [Build #361586](https://buildkite.com/elastic/kibana-pull-request/builds/361586) failed 12b0f03012a7b42dc5925c614c0449c65574e45b
* :broken_heart: [Build #361555](https://buildkite.com/elastic/kibana-pull-request/builds/361555) failed c43760d0bc7739901a1e64338a92629b7cd1e91d
* :broken_heart: [Build #361551](https://buildkite.com/elastic/kibana-pull-request/builds/361551) failed 30f5383781e1ab99d37f88e2898d51322bff8f66

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/364662","number":364662,"commit":"4599facf23803eaaca80ccfc74e6abe27f50e116"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/361604","number":361604,"commit":"d43f4d5ccce1498203c95999f4ac36f98f3a10db"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/361586","number":361586,"commit":"12b0f03012a7b42dc5925c614c0449c65574e45b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/361555","number":361555,"commit":"c43760d0bc7739901a1e64338a92629b7cd1e91d"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/361551","number":361551,"commit":"30f5383781e1ab99d37f88e2898d51322bff8f66"}],"number":364662}
buildkite-pr-comment-->

mattkime committed:
use correct user for tests

mattkime committed:
Merge branch 'index_mgmnt_playwright' of github.com:elastic/kibana into index_mgmnt_playwright

mattkime commented:
/ci

mattkime committed:
fix FTR tests

dmlemeshko commented on x-pack/platform/plugins/shared/index_management/test/scout/ui/tests/home_page.spec.ts:
```diff
@@ -0,0 +1,110 @@
+      await esClient.indices.delete({ index: testIndexName });
+    } catch (e: any) {
+      log.debug(`Index cleanup failed for ${testIndexName}: ${e.message}`);
+    }
```
_question_: having it in the test might not work if test fails, right? Do you want to move to `afterEach` hook?

kapral18 reviewed (commented):
-

kapral18 reviewed (commented):
Great job with the migration. Left a few comments.

mattkime committed:
fix FTR, improve scout tests

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/365488)
* Commit: fbb31c422fad72f60ea0ee5df3cb9022b8e83475

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/365488#019abeed-a081-469e-955b-d4d217b5d72a)

### History
* :broken_heart: [Build #364708](https://buildkite.com/elastic/kibana-pull-request/builds/364708) failed c75d6aa0f00a11e01d04ff45086bc973d6b61c71
* :broken_heart: [Build #364684](https://buildkite.com/elastic/kibana-pull-request/builds/364684) failed a577c5562c477b8af7169bfe331f34341f5a47ea
* :broken_heart: [Build #364662](https://buildkite.com/elastic/kibana-pull-request/builds/364662) failed 4599facf23803eaaca80ccfc74e6abe27f50e116
* :green_heart: [Build #361604](https://buildkite.com/elastic/kibana-pull-request/builds/361604) succeeded d43f4d5ccce1498203c95999f4ac36f98f3a10db
* :broken_heart: [Build #361586](https://buildkite.com/elastic/kibana-pull-request/builds/361586) failed 12b0f03012a7b42dc5925c614c0449c65574e45b

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/365488","number":365488,"commit":"fbb31c422fad72f60ea0ee5df3cb9022b8e83475"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/364708","number":364708,"commit":"c75d6aa0f00a11e01d04ff45086bc973d6b61c71"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/364684","number":364684,"commit":"a577c5562c477b8af7169bfe331f34341f5a47ea"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/364662","number":364662,"commit":"4599facf23803eaaca80ccfc74e6abe27f50e116"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/361604","number":361604,"commit":"d43f4d5ccce1498203c95999f4ac36f98f3a10db"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/361586","number":361586,"commit":"12b0f03012a7b42dc5925c614c0449c65574e45b"}],"number":365488}
buildkite-pr-comment-->

mattkime committed:
type fix

mattkime committed:
scout test fixes

mattkime committed:
Merge branch 'main' into index_mgmnt_playwright

mattkime committed:
fix scout test

mattkime committed:
Merge branch 'index_mgmnt_playwright' of github.com:elastic/kibana into index_mgmnt_playwright

mattkime committed:
type fix, scout test fix

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/366091)
* Commit: ec76d4a6a3afb66c798d6aa4f22913f79fa6e697

### Failed CI Steps
* [Store Cache for build](https://buildkite.com/elastic/kibana-pull-request/builds/366091#019ac1e3-b142-49ba-aa30-5e7716533df7)

### History
* :broken_heart: [Build #366072](https://buildkite.com/elastic/kibana-pull-request/builds/366072) failed ef6471892ccf2c75ed29d870f698faca07d24193
* :broken_heart: [Build #366038](https://buildkite.com/elastic/kibana-pull-request/builds/366038) failed e2a58f52ef9ce2cd2cfcd13a8b08f29ede401a33
* :broken_heart: [Build #365504](https://buildkite.com/elastic/kibana-pull-request/builds/365504) failed 40a24de6f1961447c77284b0cb13abad52b91fdb
* :broken_heart: [Build #365488](https://buildkite.com/elastic/kibana-pull-request/builds/365488) failed fbb31c422fad72f60ea0ee5df3cb9022b8e83475
* :broken_heart: [Build #364708](https://buildkite.com/elastic/kibana-pull-request/builds/364708) failed c75d6aa0f00a11e01d04ff45086bc973d6b61c71
* :broken_heart: [Build #364684](https://buildkite.com/elastic/kibana-pull-request/builds/364684) failed a577c5562c477b8af7169bfe331f34341f5a47ea

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366091","number":366091,"commit":"ec76d4a6a3afb66c798d6aa4f22913f79fa6e697"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366072","number":366072,"commit":"ef6471892ccf2c75ed29d870f698faca07d24193"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366038","number":366038,"commit":"e2a58f52ef9ce2cd2cfcd13a8b08f29ede401a33"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/365504","number":365504,"commit":"40a24de6f1961447c77284b0cb13abad52b91fdb"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/365488","number":365488,"commit":"fbb31c422fad72f60ea0ee5df3cb9022b8e83475"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/364708","number":364708,"commit":"c75d6aa0f00a11e01d04ff45086bc973d6b61c71"}],"number":366091}
buildkite-pr-comment-->

mattkime committed:
fix add button ref

mattkime committed:
fix functional test

mattkime committed:
test fixes

elasticmachine commented:
## :broken_heart: Build Failed
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/366152)
* Commit: adbf566fe7624cf1cf618d1e527d55444c7e7bda

### Failed CI Steps
* [Linting (with types)](https://buildkite.com/elastic/kibana-pull-request/builds/366152#019ac424-dcc8-4b73-a495-4bae39ff1ca7)

### History
* :broken_heart: [Build #366144](https://buildkite.com/elastic/kibana-pull-request/builds/366144) failed eca0e255df849cb1998fe28b05b50825cceaf250
* :broken_heart: [Build #366115](https://buildkite.com/elastic/kibana-pull-request/builds/366115) failed cb982b538da07e268202765a1d64821b7d57e2b8
* :broken_heart: [Build #366095](https://buildkite.com/elastic/kibana-pull-request/builds/366095) failed ec76d4a6a3afb66c798d6aa4f22913f79fa6e697
* :broken_heart: [Build #366091](https://buildkite.com/elastic/kibana-pull-request/builds/366091) failed ec76d4a6a3afb66c798d6aa4f22913f79fa6e697
* :broken_heart: [Build #366072](https://buildkite.com/elastic/kibana-pull-request/builds/366072) failed ef6471892ccf2c75ed29d870f698faca07d24193

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366152","number":366152,"commit":"adbf566fe7624cf1cf618d1e527d55444c7e7bda"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366144","number":366144,"commit":"eca0e255df849cb1998fe28b05b50825cceaf250"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366115","number":366115,"commit":"cb982b538da07e268202765a1d64821b7d57e2b8"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366095","number":366095,"commit":"ec76d4a6a3afb66c798d6aa4f22913f79fa6e697"},{"buildStatus":{"state":"running","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366091","number":366091,"commit":"ec76d4a6a3afb66c798d6aa4f22913f79fa6e697"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366072","number":366072,"commit":"ef6471892ccf2c75ed29d870f698faca07d24193"}],"number":366152}
buildkite-pr-comment-->

mattkime committed:
add missing await

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/366164)
* Commit: c950dc4137f3b73f3ca282bd7813cbba0351b1af

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/0916afc68819653198f267590162ec2021e113d9) | [after](https://github.com/elastic/kibana/commit/c950dc4137f3b73f3ca282bd7813cbba0351b1af) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 707.8KB | 708.0KB | +219.0B |

<details><summary>Unknown metric groups</summary>

#### ESLint disabled in files
| id | [before](https://github.com/elastic/kibana/commit/0916afc68819653198f267590162ec2021e113d9) | [after](https://github.com/elastic/kibana/commit/c950dc4137f3b73f3ca282bd7813cbba0351b1af) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 3 | 4 | +1 |

#### ESLint disabled line counts
| id | [before](https://github.com/elastic/kibana/commit/0916afc68819653198f267590162ec2021e113d9) | [after](https://github.com/elastic/kibana/commit/c950dc4137f3b73f3ca282bd7813cbba0351b1af) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 16 | 17 | +1 |

#### Total ESLint disabled count
| id | [before](https://github.com/elastic/kibana/commit/0916afc68819653198f267590162ec2021e113d9) | [after](https://github.com/elastic/kibana/commit/c950dc4137f3b73f3ca282bd7813cbba0351b1af) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 19 | 21 | +2 |


</details>


### History
* :broken_heart: [Build #366152](https://buildkite.com/elastic/kibana-pull-request/builds/366152) failed adbf566fe7624cf1cf618d1e527d55444c7e7bda
* :broken_heart: [Build #366144](https://buildkite.com/elastic/kibana-pull-request/builds/366144) failed eca0e255df849cb1998fe28b05b50825cceaf250
* :broken_heart: [Build #366115](https://buildkite.com/elastic/kibana-pull-request/builds/366115) failed cb982b538da07e268202765a1d64821b7d57e2b8
* :broken_heart: [Build #366095](https://buildkite.com/elastic/kibana-pull-request/builds/366095) failed ec76d4a6a3afb66c798d6aa4f22913f79fa6e697
* :broken_heart: [Build #366091](https://buildkite.com/elastic/kibana-pull-request/builds/366091) failed ec76d4a6a3afb66c798d6aa4f22913f79fa6e697
* :broken_heart: [Build #366072](https://buildkite.com/elastic/kibana-pull-request/builds/366072) failed ef6471892ccf2c75ed29d870f698faca07d24193

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366164","number":366164,"commit":"c950dc4137f3b73f3ca282bd7813cbba0351b1af"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366152","number":366152,"commit":"adbf566fe7624cf1cf618d1e527d55444c7e7bda"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366144","number":366144,"commit":"eca0e255df849cb1998fe28b05b50825cceaf250"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366115","number":366115,"commit":"cb982b538da07e268202765a1d64821b7d57e2b8"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366095","number":366095,"commit":"ec76d4a6a3afb66c798d6aa4f22913f79fa6e697"},{"buildStatus":{"state":"running","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366091","number":366091,"commit":"ec76d4a6a3afb66c798d6aa4f22913f79fa6e697"}],"number":366164}
buildkite-pr-comment-->

mattkime committed:
Merge branch 'main' into index_mgmnt_playwright

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/366337)
* Commit: 253975a8fd3e9665b425568088d1e81a3c532dd3

### Failed CI Steps
* [src/platform/test/functional/apps/visualize/replaced_vislib_chart_types/config.ts](https://buildkite.com/elastic/kibana-pull-request/builds/366337#019ac589-6279-481d-ac61-0015943ce099)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/366337#019ac589-6279-481d-ac61-0015943ce099) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/366337/jobs/019ac589-6279-481d-ac61-0015943ce099/artifacts/019ac5aa-93e9-40be-9757-a350da3b29ec) src/platform/test/functional/apps/visualize/replaced_vislib_chart_types/config.ts / visualize app - new charts library visualize Timelion visualization expression typeahead dynamic suggestions for argument values .es() should show field suggestions for timefield argument when index pattern set

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/7e88dfeb4375c566241bd70eab3aac57003bd41c) | [after](https://github.com/elastic/kibana/commit/253975a8fd3e9665b425568088d1e81a3c532dd3) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 708.0KB | 708.2KB | +219.0B |

<details><summary>Unknown metric groups</summary>

#### ESLint disabled in files
| id | [before](https://github.com/elastic/kibana/commit/7e88dfeb4375c566241bd70eab3aac57003bd41c) | [after](https://github.com/elastic/kibana/commit/253975a8fd3e9665b425568088d1e81a3c532dd3) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 3 | 4 | +1 |

#### ESLint disabled line counts
| id | [before](https://github.com/elastic/kibana/commit/7e88dfeb4375c566241bd70eab3aac57003bd41c) | [after](https://github.com/elastic/kibana/commit/253975a8fd3e9665b425568088d1e81a3c532dd3) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 16 | 17 | +1 |

#### Total ESLint disabled count
| id | [before](https://github.com/elastic/kibana/commit/7e88dfeb4375c566241bd70eab3aac57003bd41c) | [after](https://github.com/elastic/kibana/commit/253975a8fd3e9665b425568088d1e81a3c532dd3) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 19 | 21 | +2 |


</details>


### History
* :green_heart: [Build #366164](https://buildkite.com/elastic/kibana-pull-request/builds/366164) succeeded c950dc4137f3b73f3ca282bd7813cbba0351b1af
* :broken_heart: [Build #366152](https://buildkite.com/elastic/kibana-pull-request/builds/366152) failed adbf566fe7624cf1cf618d1e527d55444c7e7bda
* :broken_heart: [Build #366144](https://buildkite.com/elastic/kibana-pull-request/builds/366144) failed eca0e255df849cb1998fe28b05b50825cceaf250
* :broken_heart: [Build #366115](https://buildkite.com/elastic/kibana-pull-request/builds/366115) failed cb982b538da07e268202765a1d64821b7d57e2b8
* :broken_heart: [Build #366095](https://buildkite.com/elastic/kibana-pull-request/builds/366095) failed ec76d4a6a3afb66c798d6aa4f22913f79fa6e697
* :broken_heart: [Build #366091](https://buildkite.com/elastic/kibana-pull-request/builds/366091) failed ec76d4a6a3afb66c798d6aa4f22913f79fa6e697

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366337","number":366337,"commit":"253975a8fd3e9665b425568088d1e81a3c532dd3"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366164","number":366164,"commit":"c950dc4137f3b73f3ca282bd7813cbba0351b1af"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366152","number":366152,"commit":"adbf566fe7624cf1cf618d1e527d55444c7e7bda"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366144","number":366144,"commit":"eca0e255df849cb1998fe28b05b50825cceaf250"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366115","number":366115,"commit":"cb982b538da07e268202765a1d64821b7d57e2b8"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366095","number":366095,"commit":"ec76d4a6a3afb66c798d6aa4f22913f79fa6e697"}],"number":366337}
buildkite-pr-comment-->

kapral18 committed:
Merge branch 'main' into index_mgmnt_playwright

kapral18 committed:
fix: remove debug comment and clarify test behavior

- Removed commented-out page.pause() debug code
- Added clarifying comment explaining why addButton click was removed:
  the Create Field form is already open by default when mappings are empty

kapral18 reviewed (commented):
-

kapral18 commented on x-pack/platform/plugins/shared/index_management/test/scout/ui/fixtures/page_objects/index_management_page.ts:
```diff
@@ -0,0 +1,143 @@
+    await saveButton.waitFor({ state: 'visible' });
+    await saveButton.click();
+    // Wait for modal to close
+    await saveButton.waitFor({ state: 'hidden', timeout: 30000 });
```
I checked the ESLint config and Scout implementation:

- The `playwright/no-wait-for-selector` rule targets `page.waitForSelector()`
- Matt's code uses `locator.waitFor()` which is Playwright's recommended pattern
- Scout's `testSubj` helper exposes `waitForSelector` (proxied to page) but not `waitFor`

The current code is correct - `locator.waitFor()` is idiomatic Playwright and not flagged by the linter.

dmlemeshko commented on x-pack/platform/plugins/shared/index_management/test/scout/ui/tests/index_template_wizard.spec.ts:
```diff
@@ -0,0 +1,128 @@
+
+    // Click Create template
+    await pageObjects.indexManagement.clickNextButton();
+  });
```
_question_: do we really need to repetitively start fresh browser context and repeat UI flow from scratch?

It would be more efficient and pretty much the same outcome to switch from `test` to `test.step` because if  `await pageObjects.indexManagement.indexTemplateWizard.completeStepOne();` doesn't work, there is no point of running most of `test blocks` here. Thoughts?

kapral18 replied:
Agreed. Thanks for catching this.

dmlemeshko commented on x-pack/platform/plugins/shared/index_management/test/scout/ui/tests/index_template_wizard_preview.spec.ts:
```diff
@@ -0,0 +1,89 @@
+    const templatePreview = await page.testSubj.locator('simulateTemplatePreview').textContent();
+    expect(templatePreview).not.toContain('error');
+
+    await page.testSubj.locator('closeDetailsButton').click();
```
for better readability I suggest splitting logic into logical steps with `test.step`, it will be displayed in the report as steps as well

kapral18 committed:
refactor: use test.step for wizard tests and EuiFieldTextWrapper

- Refactor index_template_wizard.spec.ts to use test.step instead of
  separate test blocks for better efficiency and failure reporting
- Refactor index_template_wizard_preview.spec.ts to use test.step
  for improved readability
- Use EuiFieldTextWrapper in indexTemplateWizard.completeStepOne

kapral18 commented on x-pack/platform/plugins/shared/index_management/test/scout/ui/fixtures/page_objects/index_management_page.ts:
```diff
@@ -0,0 +1,134 @@
+      await this.page.testSubj
+        .locator('indexPatternsField')
+        .locator('input')
+        .fill('test-index-pattern');
```
Done! Updated `indexTemplateWizard.completeStepOne` to use `EuiFieldTextWrapper`.

kibanamachine commented:
## Flaky Test Runner Stats
### 🟠 Some tests failed. - [kibana-flaky-test-suite-runner#9958](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/9958)
[❌] x-pack/platform/plugins/shared/index_management/test/scout/ui/playwright.config.ts: 0/25 tests passed.

[see run history](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds?branch=refs%2Fpull%2F242825%2Fhead)

kapral18 committed:
refactor: use web-first assertions and EUI wrappers per best practices

- Replace waitFor({ state: 'hidden' }) with expect().toBeHidden()
- Replace waitFor({ state: 'visible' }) with expect().toBeVisible()
- Change indexLinkVisible to indexLink returning locator for caller assertions
- Update test to use web-first assertion for index visibility check
- Use EuiFieldTextWrapper for text input fields in mappings and preview tests

kapral18 committed:
Merge branch 'main' into index_mgmnt_playwright

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/368036)
* Commit: 4a3ab2b6608422ee777b1af389693b7b522f4ce9

### Failed CI Steps
* [FTR Configs #28](https://buildkite.com/elastic/kibana-pull-request/builds/368036#019ae1a3-9abd-471a-a5a7-e2f78be10131)
* [FTR Configs #63](https://buildkite.com/elastic/kibana-pull-request/builds/368036#019ae1a3-9ae8-44b5-a3ba-039607231093)
* [FTR Configs #120](https://buildkite.com/elastic/kibana-pull-request/builds/368036#019ae1a3-9b26-48ca-831a-e21c7d67f1c3)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/368036#019ae1a3-9abd-471a-a5a7-e2f78be10131) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/368036/jobs/019ae1a3-9abd-471a-a5a7-e2f78be10131/artifacts/019ae1c7-b116-455a-bd60-1b0656673323) FTR Configs #<span></span>28 / Agent Builder agents Edit agent should edit agent name
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/368036#019ae1a3-9b26-48ca-831a-e21c7d67f1c3) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/368036/jobs/019ae1a3-9b26-48ca-831a-e21c7d67f1c3/artifacts/019ae1b2-0a61-4f3a-bc7a-2657d5608feb) FTR Configs #<span></span>120 / discover/tabs duplication of a tab should restore the previous app and global states
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/368036#019ae1a3-9ae8-44b5-a3ba-039607231093) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/368036/jobs/019ae1a3-9ae8-44b5-a3ba-039607231093/artifacts/019ae1c4-2063-4142-815b-c4b815b1520a) FTR Configs #<span></span>63 / Entity Analytics - Privilege Monitoring @<span></span>ess @<span></span>serverless @<span></span>skipInServerlessMKI Entity Privilege Monitoring Engine Plain Index Sync Plain index sync should not create duplicate users

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/019e4e67b46c8680f14988ea9407140434ec30bc) | [after](https://github.com/elastic/kibana/commit/4a3ab2b6608422ee777b1af389693b7b522f4ce9) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 719.5KB | 719.7KB | +219.0B |

<details><summary>Unknown metric groups</summary>

#### ESLint disabled in files
| id | [before](https://github.com/elastic/kibana/commit/019e4e67b46c8680f14988ea9407140434ec30bc) | [after](https://github.com/elastic/kibana/commit/4a3ab2b6608422ee777b1af389693b7b522f4ce9) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 3 | 4 | +1 |

#### ESLint disabled line counts
| id | [before](https://github.com/elastic/kibana/commit/019e4e67b46c8680f14988ea9407140434ec30bc) | [after](https://github.com/elastic/kibana/commit/4a3ab2b6608422ee777b1af389693b7b522f4ce9) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 16 | 17 | +1 |

#### Total ESLint disabled count
| id | [before](https://github.com/elastic/kibana/commit/019e4e67b46c8680f14988ea9407140434ec30bc) | [after](https://github.com/elastic/kibana/commit/4a3ab2b6608422ee777b1af389693b7b522f4ce9) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 19 | 21 | +2 |


</details>


### History
* :broken_heart: [Build #367860](https://buildkite.com/elastic/kibana-pull-request/builds/367860) failed 968d9f426f72f559ddffd55f6503498f881f7da7
* :broken_heart: [Build #367764](https://buildkite.com/elastic/kibana-pull-request/builds/367764) failed f022395584eedbd190146692c93891d89654d84c
* :green_heart: [Build #367420](https://buildkite.com/elastic/kibana-pull-request/builds/367420) succeeded 21af96e3c823ce440a5ce261dd27ff943f17a401
* :green_heart: [Build #367262](https://buildkite.com/elastic/kibana-pull-request/builds/367262) succeeded 732732bb43e2acfcdc1740343413c82b77e10154
* :yellow_heart: [Build #366337](https://buildkite.com/elastic/kibana-pull-request/builds/366337) was flaky 253975a8fd3e9665b425568088d1e81a3c532dd3
* :green_heart: [Build #366164](https://buildkite.com/elastic/kibana-pull-request/builds/366164) succeeded c950dc4137f3b73f3ca282bd7813cbba0351b1af

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/368036","number":368036,"commit":"4a3ab2b6608422ee777b1af389693b7b522f4ce9"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/367860","number":367860,"commit":"968d9f426f72f559ddffd55f6503498f881f7da7"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/367764","number":367764,"commit":"f022395584eedbd190146692c93891d89654d84c"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/367420","number":367420,"commit":"21af96e3c823ce440a5ce261dd27ff943f17a401"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/367262","number":367262,"commit":"732732bb43e2acfcdc1740343413c82b77e10154"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366337","number":366337,"commit":"253975a8fd3e9665b425568088d1e81a3c532dd3"}],"number":368036}
buildkite-pr-comment-->
